diff --git a/node_modules/vue-echarts/dist/index.esm.js b/node_modules/vue-echarts/dist/index.esm.js
index 3ec3ce0..c9e8af8 100644
--- a/node_modules/vue-echarts/dist/index.esm.js
+++ b/node_modules/vue-echarts/dist/index.esm.js
@@ -1,5 +1,5 @@
 import { watch, isRef, unref, inject, computed, watchEffect, Vue2, defineComponent, shallowRef, toRefs, getCurrentInstance, onMounted, onBeforeUnmount, h, nextTick } from 'vue-demi';
-import { throttle, init } from 'echarts/core';
+import { throttle, init } from 'echarts/core.js';
 import { addListener, removeListener } from 'resize-detector';
 
 /******************************************************************************
diff --git a/node_modules/vue-echarts/package.json b/node_modules/vue-echarts/package.json
index 7e46d29..dd07c39 100644
--- a/node_modules/vue-echarts/package.json
+++ b/node_modules/vue-echarts/package.json
@@ -87,5 +87,13 @@
     "build:demo": "vue-cli-service build",
     "docs": "node ./scripts/docs.js",
     "postinstall": "node ./scripts/postinstall.js"
+  },
+  "type": "module",
+  "exports": {
+    ".": {
+      "require": "./dist/index.cjs.js",
+      "import": "./dist/index.esm.js",
+      "types": "./dist/index.d.ts"
+    }
   }
 }
